packages/odie-client/src/hooks/use-auto-scroll.ts

```diff
@@ -37,9 +37,15 @@ export const useAutoScroll = (
 		debounceTimeoutIdRef.current = setTimeout( () => {
 			debounceTimeoutRef.current = 0;
 			requestAnimationFrame( () => {
-				const messages = messagesContainerRef.current?.querySelectorAll(
-					'[data-is-message="true"],.odie-chatbox__action-message'
-				);
+				const messages =
+					messagesContainerRef &&
+					'current' in messagesContainerRef &&
```

Comment:
Is there a scenario where `messagesContainerRef` can not be a ref object with the `current` property?